org.mersenne.mprime.metainfo.xml

```diff
@@ -16,9 +17,12 @@
   <url type="homepage">https://www.mersenne.org</url>
   <developer_name>George Woltman</developer_name>
   <screenshots>
-          <screenshot>https://3.bp.blogspot.com/-CAWPIcQ9Qx8/T6vzdHj-B2I/AAAAAAAAAPY/6A_t46hl5mo/s1600/Screenshot+from+2012-05-10+19:35:57.png</screenshot>
+    <screenshot type="default">
+    <image type="source">https://3.bp.blogspot.com/-CAWPIcQ9Qx8/T6vzdHj-B2I/AAAAAAAAAPY/6A_t46hl5mo/s1600/Screenshot+from+2012-05-10+19:35:57.pn</image>
+    <caption>Main screen</caption>
```

Comment:
You have to close the beginning screenshot tag after this caption `</screenshot>`

Reply:
Thank you!